places.map

```diff
@@ -168,7 +196,7 @@ LAYER
 #endif
 #if _display_hamlets == 1
  CLASS
-  EXPRESSION {suburb,hamlet,locality}
```

Comment:
I removed this on purpose since we found this to be (mis-)used for several kinds of localities which should not have the same level of importance as settlements - e.g. have a look at http://overpass-turbo.eu/s/lkL
If you think locality is a tag important enough to display I can update this patch to include a separate _display_locality rule.
Thanks for the fast merges by the way :-)